pkg/config/config.go

```diff
@@ -57,30 +57,35 @@ func (configFile *ConfigFile) WriteYamlConfig() {
 
 //ReadConfig read the yaml config file
 func (configFile *ConfigFile) ReadYamlConfig() (error) {
-	filename, _ := filepath.Abs(configFile.Path)
-
-	yamlFile, err := ioutil.ReadFile(filename)
-	if err != nil {
-		return err
-	}
-
-	// If the config file are empty, make a new servers list
-	if len(yamlFile) == 0 {
-		log.Infoln("Configfile named ", filename, " are actually empty. Init servers list in-memory")
+	// If the config does not exist
+	if _, err := os.Stat(configFile.Path); os.IsNotExist(err) {
+		log.Infoln("Configfile in does not exist . Init servers list in-memory")
 		emptyServers := make(server.Servers)
 		configFile.Servers = &emptyServers
-
-	//If the config file are not empty, do more checks to load his content
+		//If the config file are not empty, do more checks to load his content
```

Comment:
Suggestion: `// If the config file is not empty, do more checks to load its content.`

On devrait toujours mettre un espace après `//` quand c'est des commentaires.

Reply:
Note: Je suggère de checker si le fichier existe dans le `if` au lieu de checker s'il existe pas et de faire du traitement dans le `else`. La condition plus encline d'être vraie et qu'on souhaite qui soit réalisée devrait toujours être faite dans le `if`, pas dans le `else`. Le `else` devrait être réservé pour les erreurs (cas négatifs).